flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/plan/ExpressionReductionRulesTest.scala

```diff
@@ -510,3 +528,14 @@ object DeterministicNullFunc extends ScalarFunction {
   def eval(): String = null
   override def isDeterministic = true
 }
+
+object MockedPythonUDFWithoutArguments extends ScalarFunction {
+
+  override def getLanguage: FunctionLanguage = FunctionLanguage.PYTHON
+
+  def eval(): Long = {
```

Comment:
Simplify the eval to `def eval(): Long = 1L` directly? and change the class name to `DeterministicPythonFunc`.